src/Opserver.Core/Data/Exceptions/ExceptionStore.cs

```diff
@@ -171,6 +172,36 @@ public class SearchParams
             public Guid? StartAt { get; set; }
             public ExceptionSorts Sort { get; set; }
             public Guid? Id { get; set; }
+            public string Url { get; set; }
+            public string Host { get; set; }
+
+            public bool IsNonDefault
+            {
+                get
+                {
+                    if (Host.HasValue()) return true;
+                    if (Url.HasValue()) return true;
+                    if (StartDate != null) return true;
+                    if (EndDate != null) return true;
+                    return false;
+                }
+            }
+
+            public string NormalizedStartDate
+            {
+                get
+                {
+                    return NormalizeDateTime(StartDate);
+                }
+            }
+
+            public string NormalizedEndDate
+            {
+                get
+                {
+                    return NormalizeDateTime(EndDate);
+                }
+            }
```

Comment:
Can use the new goodness here:
```suggestion
            public string NormalizedStartDate => NormalizeDateTime(StartDate);
            public string NormalizedEndDate => NormalizeDateTime(EndDate);
```

Reply:
Done